responsibleai/responsibleai/modelanalysis/model_analysis.py

```diff
@@ -216,6 +223,12 @@ def _validate_model_analysis_input_parameters(
                     'The serializer should be serializable via pickle')
 
         if isinstance(train, pd.DataFrame) and isinstance(test, pd.DataFrame):
+            if test.shape[0] > maximum_rows_for_test:
+                raise UserConfigValidationException(
```

Comment:
Can you add the number of rows in the test as well?

Reply:
Good idea. Done.